### ISSUE TITLE
Merge in v0.31.0 from glubsy/h5ai

### ISSUE THREAD

**Issue by Kage** [HackThisSite/h5ai#24]
Bring `main` branch current with v0.31.0 released in https://github.com/glubsy/h5ai

**Comment by nephatrine** [comment]
Just want to chime in that it it would be great to have those v0.31.0 updates but on a fork that isn't completely dead.

**Comment by Kage** [comment]
@nephatrine Indeed, however as noted in the label for this issue, help is needed to bring this active and current.  I setup this repository as a place where multiple people can maintain it, should I get the help to do so.  However I don't have enough time nor enough experience with the JS side of things to maintain it alone.

If you have the time, I'd be delighted to review and merge in any PRs you'd like to submit.
